### DISCUSSION TITLE
Option to disable app list

### DISCUSSION THREAD

**Discussion by xManan** [tanujnotes/Olauncher#272] [Ideas]
Option to turn off swipe up app list

**Comment by tanujnotes** [comment]
No plan to do that.

**Comment by CreativeCodeCat** [comment]
> Option to turn off swipe up app list

[mLauncher](https://github.com/HeCodes2Much/mLauncher) which is a fork of [OlauncherCF](https://github.com/OlauncherCF/OlauncherCF) which is also a fork of this project, can do this. hope to add it to OlauncherCF itself one day but who knows.
